src/components/SidePanel/NavGroup/FilterNavGroup/index.tsx

```diff
@@ -57,26 +65,49 @@ const FilterNavGroup: React.FC<FilterNavGroupProps> = ({
   };
 
   return (
-    <BaseNavGroup
-      data-cy="filters"
-      defaultMessage="No filters have been applied."
-      glyph="Filter"
-      items={filters}
-      navGroupTitle="Filters"
-    >
-      {filters.map((filter) => (
-        <FilterWrapper key={filter.name} data-cy={`filter-${filter.name}`}>
-          <FilterGroup
-            deleteFilter={deleteFilter}
-            editFilter={editFilter}
-            filter={filter}
-          />
-        </FilterWrapper>
-      ))}
-    </BaseNavGroup>
+    <>
+      <ApplyFiltersModal open={open} setOpen={setOpen} />
+      <BaseNavGroup
+        additionalHeaderText={
+          // TODO: Unhide in EVG-19897.
+          isProduction ? null : (
+            <ModalTrigger onClick={() => setOpen(true)}>
```

Comment:
Since this span is now a button I think it would benefit from some accessibility enhancements. Normally this would have been caught by our linters but styled components prevent that.  
https://www.w3.org/WAI/ARIA/apg/patterns/button/examples/button/